### PR TITLE
Add ServiceNow Auth guide

### PR DESCRIPTION
**Pull request by indrapaul24**

### CODE REVIEW

File: authentication-guides/servicenow/README.md [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+2. If you’re using a production or implementation tenant owned by your organization
+    1. Create a service account user (`username` & `password`), note it down
+    2. Your instance name is the prefix of your URL: `https://{{instance_name}}.service-now.com`
+
+## Step 2: Grant the appropriate permissions for your service account
+
+Follow steps in our help site to grant the necessary roles/permissions required to execute the API calls:
+
+[https://help.moveworks.com/docs/servicenow-access-requirements#service-account-requirements--permissions](https://help.moveworks.com/docs/servicenow-access-requirements#service-account-requirements--permissions)
+
```
Comment (ajaymerchia) [inline authentication-guides/servicenow/README.md:40]: Swap for https://docs.servicenow.com/bundle/tokyo-platform-administration/page/administer/roles/reference/r_BaseSystemRoles.html

File: authentication-guides/servicenow/Authentication Tutorial ServiceNow v2 bf9283817e704a4385a4c5e8ffe8859e/Screenshot_2024-03-07_at_8.35.19_AM.png
Comment (ajaymerchia) [inline authentication-guides/servicenow/Authentication Tutorial ServiceNow v2 bf9283817e704a4385a4c5e8ffe8859e/Screenshot_2024-03-07_at_8.35.19_AM.png:1]: Redact Client ID

**Review by ajaymerchia — CHANGES_REQUESTED** [review]

File: authentication-guides/servicenow/Authentication Tutorial ServiceNow v2 bf9283817e704a4385a4c5e8ffe8859e/Screenshot_2024-03-07_at_8.35.19_AM.png
Comment (sarthaksrinivas) [reply authentication-guides/servicenow/Authentication Tutorial ServiceNow v2 bf9283817e704a4385a4c5e8ffe8859e/Screenshot_2024-03-07_at_8.35.19_AM.png:1]: Screenshot_2024-03-07_at_8 35 19_AM
Indrashis could you re-upload this image?

Comment (sarthaksrinivas) [reply authentication-guides/servicenow/Authentication Tutorial ServiceNow v2 bf9283817e704a4385a4c5e8ffe8859e/Screenshot_2024-03-07_at_8.35.19_AM.png:1]: image

**Review by sarthaksrinivas — APPROVED** [review]

**Review by ajaymerchia — APPROVED** [review]

**Pull request merged**